### DISCUSSION TITLE
HTR 5860

### DISCUSSION THREAD

**Discussion by TimEyre** [mvdwetering/yamaha_ynca#363] [General]
We have a old Yamaha HTR 5860. Lost the remote years ago, just getting a new one for Christmas and am also looking to integrate into HA. Just got a IR Blaster Smart Life and looking to use it for the controller from the computer and HA. Don't know where to go from here, Sorry. 
Thanks,
Tim

**Comment by mvdwetering** [comment]
I am not sure what you are looking for.

I looked at the manual for the HTR-5860, but it seems to have no network connectivy or a serial port. That means this integration will not be of any help because it can't connect to the receiver.

I assume you are then trying to send the remote control codes from HA to the IR Blaster. I don't know that IR Blaster and there seems to be no documentation in the Home Assistant website (I think Smart Life is Tuya).

IR remote control codes can probably be found on the internet. I did not find specific ones for HTR-5860, but I assume they are similar for all receivers.

You will probably have more success asking these more general questions on the Home Assistant forums/Discord/Reddit or other community where there are more people.
